region/brinstar/red/Hellway.json

```diff
@@ -125,7 +125,7 @@
       "notable": true,
       "requires": [
         "canTrickyJump",
-        "h_canFrozenEnemyRunway",
+        "h_canTrickyFrozenEnemyRunway",
```

Comment:
Should there be an option for using a Zeela now?  Here or at 2.

Reply:
Definitely. Good catch.